discordbot/fetch_dives.py

```diff
@@ -0,0 +1,72 @@
+from ansitable import ANSITable
+
+
+def _clean_line(line):
+	return [
+		# Return every token
+		token.strip()
+		# delimited by `|`
+		for token in line.split("|")
+		# if it is not empty
+		if token
+	]
+
+def _format_warning(w):
+	# No warning, no special highlighting
+	if w == "None":
+		return "{0}".format(w)
+	# No warning known yet, highlight in white
+	elif w == "TBA":
+		return "({0})".format(w)
+	# Warning found, highlight in red
+	else:
+		return "[{0}]".format(w)
+
+
+def fetch_dives(selftext):
+	raw_text = selftext.split("\n")[4:19]
+
+	dives = {
+		"dd": {
+			"place": "",
+			"table": ""
+		},
+		"edd": {
+			"place": "",
+			"table": ""
+		}
+	}

```

Comment:
10 lines is way too much to define a simple object. For readability, this should suffice as well:
```py
dives = {
   "dd": {"place": "", "table": ""},
   "edd": {"place": "", "table": ""}
}
```